### DISCUSSION TITLE
How to import sessions

### DISCUSSION THREAD

**Discussion by 2dmartin** [sienori/Tab-Session-Manager#720] [Q&A] [answered]
On an old PC I used 'export' to save a (large) json file of my session.
On a new PC I want to import the same session.
How do I do that ?
Thanks.

**Comment by sienori** [comment, marked as answer]
You can import sessions from the "Sessions" section of the options page. Select files on the "Import Sessions".

![settings](https://user-images.githubusercontent.com/29514424/110962116-ef1cb800-8393-11eb-8bc4-ca46df0ffe88.png)
![sessions](https://user-images.githubusercontent.com/29514424/110962431-55a1d600-8394-11eb-951e-7ea5a1b64684.png)

**Comment by 2dmartin** [comment]
Ah, thank you!
I think part of the problem was that "session" does not convey quite the
meaning for me.  "Collection" might be better?

                          regards        ... David

**Comment by 2dmartin** [comment]
OK I've tried it.  Does not work.  I had about 800 tabs saved.  2 seem to
load.   It's a 7MB json file and I can see the exported data there.  But
import does not work.

                  ... David

**Reply by sienori** [reply]
Can you send me the file so I can debug it? sienori.firefox+tsm@gmail.com
(I will be very careful with your privacy)

**Comment by 2dmartin** [comment]
Would like to but I think there's far too much personal info in there.
You might be able to convince me ....

                  ... David
